profiles/common/modules/custom/nexteuropa_fields_formatters/nexteuropa_fields_formatters.module

```diff
@@ -93,10 +90,10 @@ function nexteuropa_fields_formatters_field_formatter_settings_form($field, $ins
           'light' => 'Light',
```

Comment:
`Light`, `Block` and `info` do not exist in the ecl button variant. 

So is it normal to find them in the variation options?